lib/phoenix_live_view/utils.ex

```diff
@@ -503,4 +503,26 @@ defmodule Phoenix.LiveView.Utils do
   defp flash_salt(endpoint_mod) when is_atom(endpoint_mod) do
     "flash:" <> salt!(endpoint_mod)
   end
+
+  def write_tmp_file!(filename, extension, content) do
```

Comment:
Since these functions are not reused from any other module, they should be defined in the modules that use them! :)